src/geo/ui/search/search.js

```diff
@@ -67,11 +67,13 @@ var Search = View.extend({
   },
 
   _initializeGeocoder: function () {
-    this.geocoderService = this.options.geocoderService || DEFAULT_GEOCODER;
+    const injectedGeocoderConfig = this._getGeocodingInfoFromConfig();
```

Comment:
The only thing that tingles my spidey sense is the mix of configurations.

I can get the provider from the injected config but the token can be undefined and then  I go directly for the window property... those combinations.

That won't break the code but if a configuration is wrong perhaps it makes it harder to debug.